src/module-elasticsuite-core/Model/Search/Request/RelevanceConfig/Structure/Data.php

```diff
@@ -0,0 +1,26 @@
+<?php
+/**
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade Smile ElasticSuite to newer
+ * versions in the future.
+ *
+ * @category  Smile
+ * @package   Smile\ElasticsuiteCore
+ * @author    Thomas Klein <thomasklein876@gmail.com>
+ * @copyright 2020 Smile
+ * @license   Open Software License ("OSL") v. 3.0
+ */
+declare(strict_types=1);
```

Comment:
Please put spaces around the assign operator ->
`declare(strict_types = 1);`

Reply:
Will do